roles/adv-prog-pkgs/tasks/main.yml

```diff
@@ -1,27 +1,10 @@
 ---
 # tasks file for adv-prog-pkgs
 - name: Install advanced language packages
-  apt: name={{ item }} state=latest
-  with_items:
-    - build-essential
-    - g++
-    - gcc
-    - gdb
-    - logisim
-    - valgrind
+  apt: name={{ adv_prog_pkgs_adv_lang }} state=latest
```

Comment:
Nice! I like that these have become one-line entries in the config files! Thanks!